### PR TITLE
new meta

### PR DESCRIPTION
**Pull request by LouiseMedova**

### CODE REVIEW

File: syndote/.metahash [hunk 1]
```diff
@@ -0,0 +1 @@
+[176, 113, 233, 195, 233, 156, 188, 34, 55, 179, 62, 250, 235, 58, 24, 193, 216, 57, 254, 234, 174, 205, 227, 96, 204, 227, 220, 165, 129, 49, 209, 227]
```
Comment (fluiderson) [inline syndote/.metahash]: Add this file to `.gitignore`. It's only needed during building.

File: syndote/src/utils.rs [hunk 2]
```diff
@@ -64,24 +66,21 @@ pub fn sell_property(
...
-static mut SEED: u64 = 0;
 pub fn get_rolls() -> (u8, u8) {
-    let seed = unsafe {
-        SEED = SEED.wrapping_add(1);
-        SEED
-    };
-    let random = exec::random(&(exec::block_timestamp() + seed).to_be_bytes()).expect("");
+    let mut random_input: [u8; 32] = [0; 32];
+    Xoshiro128PlusPlus::seed_from_u64(exec::block_timestamp()).fill_bytes(&mut random_input);
+    let random = exec::random(random_input).expect("");
```
Comment (fluiderson) [inline syndote/src/utils.rs]: Isn't just `exec::random()` not enough? `Xoshiro128PlusPlus` will just mix up already mixed up numbers and spend extra gas.

**Review by fluiderson — APPROVED** [review]

File: syndote/src/utils.rs (hunk 2 above)
Comment (LouiseMedova) [reply syndote/src/utils.rs]: I used it for seed generation

Comment (fluiderson) [reply syndote/src/utils.rs]: If I'm not mistaken, `exec::random()` generates a seed by itself on every execution, so it's not necessary to do that inside a contract. But you can leave that for extra randomness.

Comment (LouiseMedova) [reply syndote/src/utils.rs]: you are right

**Review by bezrazli4n0 — APPROVED** [review]

**Pull request merged**